docs/resources/Guild.md

```diff
@@ -751,8 +753,22 @@ Modify attributes of a [guild member](#DOCS_RESOURCES_GUILD/guild-member-object)
 | deaf       | boolean             | whether the user is deafened in voice channels. Will throw a 400 if the user is not in a voice channel | DEAFEN_MEMBERS   |
 | channel_id | snowflake           | id of channel to move user to (if they are connected to voice)                                         | MOVE_MEMBERS     |
 
+## Modify Current Member % PATCH /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/members/@me
+
+Modifies the current member in a guild. Returns a 200 with a partial member object containing the changed data on success. Fires a [Guild Member Update](#DOCS_TOPICS_GATEWAY/guild-member-update) Gateway event.
+
+###### JSON Params
+
+| Field | Type    | Description                    | Permission      |
+| ----- | ------- | ------------------------------ | --------------- |
+| ?nick | ?string | value to set users nickname to | CHANGE_NICKNAME |
```

Comment:
Does this mean this route ONLY takes a nickname, and we'd have to use the `/members/<bot id>` for roles and such?

Reply:
It appears to only accept nickname, yes. Providing roles does nothing.

Reply:
This route also takes a `?avatar` of type `?image data` (https://discord.com/developers/docs/reference#image-data).

That field requires nitro [*not nitro classic*]. If set and user has nitro, will update the guild specific avatar.